dexter/src/main/java/com/karumi/dexter/Dexter.java

```diff
@@ -47,6 +49,24 @@ public static void initialize(Context context) {
   }
 
   /**
+   * Initializes the library.
+   *
+   * @param activity context used by Dexter. Remember to invoke {@link com.karumi.dexter.Dexter}
+   * stop method to avoid memory leaks.
+   */
+  public static void initialize(Activity activity) {
+    if (instance == null) {
+      AndroidPermissionService androidPermissionService = new AndroidPermissionService();
+      IntentProvider intentProvider = new IntentProvider();
+      instance = new DexterInstance(activity, androidPermissionService, intentProvider);
+    }
+  }
+
+  public static void stop() {
```

Comment:
I'm guessing this method is meant to let the JVM clean the activity reference. Is it possible to keep a weak reference so that the user doesn't have to explicitly call the stop method?